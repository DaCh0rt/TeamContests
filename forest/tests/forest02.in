4 392
7 7
5 6
4 9
3 4
