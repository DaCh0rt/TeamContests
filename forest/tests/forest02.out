2476.71 488130.39
